<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player_settings_dialog</class>
 <widget class="QDialog" name="player_settings_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwin.qrc">
    <normaloff>:/mainWin/logo.png</normaloff>:/mainWin/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting delay (in milliseconds):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="delaySpinBox">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>7</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>The delay will be applied also to the commands time interval (the time between executing each command). If you don't want the interval you are able remove it by putting setInterval(0) at beginning of the script.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mainwin.qrc"/>
 </resources>
 <connections/>
</ui>
